labs/idp-okta-integration/readme.md

```diff
@@ -0,0 +1,182 @@
+id: idp-okta-integration
```

Comment:
Please rename this file to README.md to be consistent with the rest of devrel

Reply:
Up to you if you want to follow the existing labs and have a lab.md and a README.md file or but both in the same.